src/containers/MyNdla/components/UserProfileTag.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import styled from '@emotion/styled';
+import { colors, spacing, misc } from '@ndla/core';
+import SafeLink from '@ndla/safelink';
+import { useTranslation } from 'react-i18next';
+import { Text } from '@ndla/typography';
+import { checkIfModerator } from '../../../config';
+import Avatar from './Avatar';
+import { useArenaUser } from '../arenaQueries';
+
+type UserProfileTagProps = {
+  displayName: string | undefined;
+  username: string | undefined;
+  affiliation: string | undefined;
+};
+
+const Name = styled(Text)`
+  text-decoration: underline;
+`;
+
+const UserProfileTagContainer = styled(SafeLink)`
+  display: flex;
+  gap: ${spacing.normal};
+  color: ${colors.text.primary};
+  height: fit-content;
+  width: fit-content;
+  text-decoration: none;
+  box-shadow: none;
+  &:hover {
+    [data-name='hover'] {
+      text-decoration: none;
+    }
+  }
+  padding: ${spacing.xsmall};
+`;
+
+const UserInformationContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${spacing.xxsmall};
+`;
+
+const NameAndTagContainer = styled.div`
+  display: flex;
+  gap: ${spacing.small};
+  align-items: center;
+`;
+
+const ModeratorTag = styled(Text)`
+  border-radius: ${misc.borderRadius};
+  padding: 2px ${spacing.small};
+  background-color: ${colors.brand.primary};
+  width: fit-content;
+  height: fit-content;
+  color: ${colors.white};
+`;
+
+const UserProfileTag = ({
+  displayName,
+  username,
+  affiliation,
+}: UserProfileTagProps) => {
+  const { t } = useTranslation();
+  const { arenaUser } = useArenaUser(username ?? '');
+
+  return (
+    // missing link to profile
+    <UserProfileTagContainer to="/myndla/arena/user/{username}">
```

Comment:
Trur denne må være slik for at dette skal fungere.
```suggestion
    <UserProfileTagContainer to={`/myndla/arena/user/${username}`}>
```
Ellers hadde det sikkert vore lurt å legge alle pather inn en plass, fekst routehelpers eller nokke, slik at vi slipper å endre mange plasser dersom det skulle dukke opp endringer i pathen. Men det kan kanskje tas når denne komponenten tas i bruk